multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/helpers/SystemParameters.java

```diff
@@ -36,7 +35,7 @@ public class SystemParameters {
     private final String authorizationEndpoint;
     private final String deployServiceUrl;
     private final boolean reserveTemporaryRoutes;
-    private final Supplier<String> timestampSupplier;
+    private final String timestamp;
```

Comment:
Why is this changed from a generator of a timestamp to a concrete timestamp that will be from the creation of the `SystemParameters` object?

Reply:
this is changed, so that the timestamp will be immutable for the whole deployment (it's now generated at the start of the operation in [OperationsApiServiceImpl.addServiceParameters()) - before that the timestamp was generated twice for Blue-Green processes (in CollectBLueGreenSystemParametersStep and in the CollectSystemParametersStep after continue)